### DISCUSSION TITLE
Releasing v10.3 in late August 2022? EDIT: mid September...

### DISCUSSION THREAD

**Discussion by vhelin** [vhelin/wla-dx#530] [General]
I just discovered a bug in pending calculations solver in WLALINK which was kinda nasty and after fixing it (hopefully!) I'm thinking that v10.3 should be put out pretty soon as I consider that bug to be a pretty massive. I'll try to solve a couple of other issues this weekend, but if you guys could try out the latest sources and report back if there are any new issues, that'd be great!

**Comment by lhsazevedo** [comment]
I'm not a advanced user as one could be but, seems to be working fine for me. Thank you for the quick fix! (multi-byte definitions)

**Reply by vhelin** [reply]
> I'm not a advanced user as one could be but, seems to be working fine for me. Thank you for the quick fix! (multi-byte definitions)

That was also a big fix, kind of. Also made two kinda big bug fixes yesterday. I hope I didn't break anything there, all the tests pass, but that just means that all the tests pass - code that the tests don't cover might be broken...

**Comment by vhelin** [comment]
Ok, if no new bugs are found I'll release v10.3 during 2.-4. September 2022 (next weekend).

**Comment by vhelin** [comment]
Just fixed a flaw in the .FUNCTION support, had to mod the stack calculator a bit and also I need to optimize the current implementation more and write more tests which means that v10.3 release has been postponed at least until 10.-11. September 2022...

**Comment by vhelin** [comment]
I've been making some changes this week so I'll postpone v10.3 release by one week -> 17.-18. September 2022...

**Comment by vhelin** [comment]
Ok, I'm done pushing new commits to v10.3a. If people say that this works then it'll release as v10.3...

**Comment by vhelin** [comment]
I'm reading the lack of posts here as "everything just works" and will release v10.3 before the end of this month. :) You still can post your bug reports before that and make me to fix them before the release!
